sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_azure/_token_manager.py

```diff
@@ -82,7 +82,7 @@ def get_token(self, *scopes: str, claims: Union[str, None] = None, tenant_id: Un
             access_token = credential.get_token(self.token_scope)
             self._update_token(access_token)
 
-        return self.token  # check for none is hidden in the _token_needs_update method
```

Comment:
Azure SDKs and autogenerated code are not compatible with a token as string they expect AccessToken object hence the change was made. Can we follow the same contract here ?